dissect/target/containers/hdd.py

```diff
@@ -0,0 +1,37 @@
+import io
+from pathlib import Path
+from typing import BinaryIO, Union
+
+from dissect.hypervisor import hdd
+
+from dissect.target.container import Container
+
+
+class HddContainer(Container):
+    def __init__(self, fh: Path, *args, **kwargs):
+        if hasattr(fh, "read"):
+            raise TypeError("HddContainer can only be opened by path")
+
+        self.hdd = hdd.HDD(fh)
+        self.stream = self.hdd.open()
+        super().__init__(fh, self.stream.size, *args, **kwargs)
+
+    @staticmethod
+    def detect_fh(fh: BinaryIO, original: Union[list, BinaryIO]) -> bool:
+        return False
+
+    @staticmethod
+    def detect_path(path: Path, original: Union[list, BinaryIO]) -> bool:
+        return path.suffix.lower() == ".hdd"
+
+    def read(self, length: int) -> bytes:
```

Comment:
Maybe this issue has been raised way before my involvement but... I have looked at all the containers and the read/seek/tell/close methods seem to be much alike. Isn't it an idea to move these to the parent? That would reduce the amount of code.